.mergify.yml

```
@@ -0,0 +1,11 @@
+queue_rules:
+ - name: default
+   conditions: []
+
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - base=main
```

We can expand this further.

```suggestion
    conditions:
      - base=main
      - label=s:accepted
```

---

I think this lable check can be done in github branch protection rules?

---

Not that I know. If you can reference documentation about it I prefer to skip it, because I am in favor of setting all possible rules in the branch protection rules.